### ISSUE TITLE
Make the word Affiliate customizable

### ISSUE THREAD

**Issue by michaelbeil** [AffiliateWP/AffiliateWP#2492]
A user suggested the following: 

>I have thought about "Say What" plugin, and was just wondering if there was any option out of the box. I'm not sure about the level of demands for this, but I think the option to just change the word "Affiliate" to something similar is quite helpful, and makes more personal and unique to each site. Just a thought, it would be great if you can consider this a feature request :slightly_smiling_face:.

https://secure.helpscout.net/conversation/495404201/73093?folderId=205529

**Comment by amdrew** [comment]
Could be a great option to add, saves a lot of time changing all the text strings in SayWhat. I've seen a lot of requests to change "affiliate" to something like "partner", "ambassador" and others.

**Comment by gspice** [comment]
Yes, another vote for this, wants to change the word "affiliate" everywhere in the app to "consultant".
https://secure.helpscout.net/conversation/505228250/74089?folderId=1170842
